src/snappy_device_agents/data/riverside/network-config.cfg

```diff
@@ -0,0 +1,11 @@
+#cloud-config
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    eth0:  # Your ethernet name.
+      dhcp4: no
+      addresses: [<ip>/22]
+      gateway4: 10.102.160.1
```

Comment:
None of this should be hardcoded. Instead, we should be getting it through dhcp, and we can configure a stable IP to give the system through dchp in the lab. Hardcoding this is likely to break.

Reply:
There is currently an issue with the riverside device (not likely to be fixed in a short term period). The MAC address is created randomly at every boot. As the DHCP configuration is based on it to attribute an IP lease, it blocks the correct execution of TF. I worked around that issue with a fixed IP configuration (this config)... I guess we might also be able to fix it using a port based DHCP configuration, but it would involve much more complexity in the lab setup, and I don't think this is what your team want...

Reply:
How soon are we likely to get a fix for this with the randomly changing mac address? That's something that we've seen on other devices before and generally a critical bug that gets addressed quickly. If we have to support this for a longer period of time, then I think we should just add this as a feature to the muxpi device agent to allow injecting something like this - at least for this device.